libcontainer/selinux/selinux.go

```diff
@@ -33,19 +33,29 @@ const (
 )
 
 var (
-	assignRegex           = regexp.MustCompile(`^([^=]+)=(.*)$`)
-	mcsList               = make(map[string]bool)
-	mcsLock               sync.Mutex
-	selinuxfs             = "unknown"
-	selinuxEnabled        = false // Stores whether selinux is currently enabled
-	selinuxEnabledChecked = false // Stores whether selinux enablement has been checked or established yet
+	assignRegex                   = regexp.MustCompile(`^([^=]+)=(.*)$`)
+	mcsList                       = make(map[string]bool)
+	mcsLock                       sync.Mutex
+	selinuxfs                     = "unknown"
+	selinuxEnabled                = false // Stores whether selinux is currently enabled
+	selinuxEnabledHost            = false // Stores whether selinux is currently enabled on the host
+	defProcessLabel, defFileLabel = getDefaultLabels()
 )
 
 type SELinuxContext map[string]string
 
+func init() {
+	if fs := getSelinuxMountPoint(); fs != "" {
+		if con, _ := Getcon(); con != "kernel" {
```

Comment:
That is the code we have been using all along.    I guess it could get permission denied.